### ISSUE TITLE
Weird Results After Training with Ignore Labels

### ISSUE THREAD

**Issue by giuliabaldini** [MIC-DKFZ/nnUNet#2138]
Hi there,

thank you very much for including the [training with masked regions](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/ignore_label.md) in nnUNet!

We have tried this on the [SAROS dataset](https://doi.org/10.25737/SZ96-ZG60) and although the training runs smoothly and the results on the annotated slices are good, the output segmentations are very messy. This is somehow okay for abdominal CTs, but it gets very weird with whole body CTs.

Below is an example for an abdomen CT (case_001) (ground truth on the left, prediction on the right). You can see that the segmentation is very good for the existing slice.
![image](https://github.com/MIC-DKFZ/nnUNet/assets/44327645/088a6d1d-a645-46d9-84f8-1c6fd0c85ed6)

And then, for the same scan, another slice (prediction only):
![image](https://github.com/MIC-DKFZ/nnUNet/assets/44327645/16344cdc-ddb2-45b0-af22-66fbf70e3c9b)

For a whole body (case_600), however, the prediction is much worse:
![image](https://github.com/MIC-DKFZ/nnUNet/assets/44327645/27da3dac-c29e-4ffa-ae8a-af16ebe984a0)

I attach our [conversion script](https://github.com/MIC-DKFZ/nnUNet/files/15108920/move_data_ignore.py.zip), the [training script](https://github.com/MIC-DKFZ/nnUNet/files/15108945/train_regions_ignore.sh.zip) and the [progress.png](https://github.com/MIC-DKFZ/nnUNet/assets/44327645/f2b41685-04a5-4bae-88b0-109c161fc47e) file. You can download the `Segmentation-Info_09-29-2023.csv` needed for the conversion from [here](https://www.cancerimagingarchive.net/wp-content/uploads/Segmentation-Info_09-29-2023.csv).
We have actually trained two models with different segmentations and the results are similar, so I have only posted the predictions for one of them. 

We have also successfully trained an nnUNet model in 2d and with a non-sparse version of these labels, so other approaches work well.

Do you see anything that might be wrong? 

Thank you very much!

Best,
Giulia & René

@Goku1110

**Comment by Lars-Kraemer** [comment]
Hey @giuliabaldini,

could you resolve the problem and is it still happening with the current nnUNet version?

Best,
Lars
